src/applications/mhv-secure-messaging/components/AttachmentsList.jsx

```diff
@@ -209,7 +209,7 @@ const AttachmentsList = props => {
         (attachments.length > 1 ? (
           <VaAlert
             data-testid="attachment-virus-alert"
-            aria-label={Alerts.Message.ATTACHMENT_SCAN_FAIL}
+            aria-label={Alerts.Message.MORE_THAN_ONE_ATTACHMENT_SCAN_FAIL}
```

Comment:
looks like you updated the name here, but the key in constants should be updated to match?